src/dropbox-base.js

```diff
@@ -124,6 +125,8 @@ export class DropboxBase {
     this.clientSecret = options.clientSecret;
     this.selectUser = options.selectUser;
     this.selectAdmin = options.selectAdmin;
+    this.fetch = options.fetch || fetch;
+    if (!this.fetch) { console.warn('Global fetch is deprecated and will be unsupported in a future version. Please pass fetch function as option when instantiating dropbox instance: new Dropbox({fetch})'); } // eslint-disable-line no-console
   }
```

Comment:
@mLuby Sorry if I'm reading the code wrong. Doesn't `this.fetch = options.fetch || fetch` set it to either the global fetch or the passed in one? How does `!this.fetch` evaluate to true when `options.fetch` is null?

Reply:
Whoops, nice catch, was meant to be `options.fetch`. Fixed in d42044a